File: 103-2_iOS程式設計/SwitchDemo/SwitchDemo/ViewController.m
//
//  ViewController.m
//  SwitchDemo
//
//  Created by CSIEVNUT403 on 2015/3/23.
//  Copyright (c) 2015年 CSIEVNUT403. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@property (strong, nonatomic) IBOutlet UISwitch *switcher;

@property (strong, nonatomic) IBOutlet UILabel *label;

- (IBAction)switchOnOff:(id)sender;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    [self.switcher setOn:false];
    self.label.text = @"false";
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)switchOnOff:(id)sender {
    UISwitch *switcher = (UISwitch *)sender;
//    self.label.text = [NSString stringWithFormat:@"%d", [switcher isOn]];
//    [self.label setText:[NSString stringWithFormat:@"%d", [switcher isOn]];
    
    NSLog(@"%d", [switcher isOn]);
    
    if ([switcher isOn]) {
        self.label.text = @"true";
    } else {
        self.label.text = @"false";
    }
}

@end

File: 106-2_iOS程式設計/SingleButton/SingleButton/ViewController.swift
import UIKit

class ViewController: UIViewController {
    @IBOutlet var textFieldName: UITextField! //連結姓名輸入欄位
    @IBOutlet var labelMsg: UILabel! //連結訊息欄位
    
    //建立按鈕 Touch Up Inside 事件
    @IBAction func sureClick(sender: UIButton) {
        if textFieldName.text != "" {
            labelMsg.text = (textFieldName.text)! + "，歡迎光臨！"
        } else {
            labelMsg.text = "必須輸入姓名！"
        }
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

